flash_update/views.py

```diff
@@ -105,3 +109,22 @@ class ShareFlashUpdateViewSet(
     queryset = FlashUpdateShare.objects.all()
     serializer_class = ShareFlashUpdateSerializer
     permission_class = [permissions.IsAuthenticated]
+
+
+class ExportFlashUpdateView(views.APIView):
+    permission_classes = [permissions.IsAuthenticated]
```

Comment:
Are all flash updates open to all logged-in users?

Reply:
@frozenhelium Dai??

Reply:
yep, that's the behaviour

Reply:
Sounds good.